sycl/test/reduction/reduction_usm.cpp

```diff
@@ -1,3 +1,4 @@
+// XFAIL: system-windows && accelerator
```

Comment:
```suggestion
// XFAIL: accelerator
```

Reply:
Hi @bader ,   stream.cpp test regression need to be fixed by CodeGen , reduction_usm.cpp test regression need to be  fiexd by OCLCPU/OCLFPGAEMU runtime, broadcast.cpp/broadcast_fp64.cpp tests need to be fixed by VPO vectorizer. All of them need to wait for the next release.